README.adoc

```diff
@@ -6,26 +6,29 @@ The metrics collector is a companion application to the http://cloud.spring.io/s
 It collects metrics emitted by http://cloud.spring.io/spring-cloud-stream/[Spring Cloud Stream] apps, and groups them together around the stream definition that dataflow used to deploy them.
 
 All of the OOB http://cloud.spring.io/spring-cloud-stream-app-starters/[Spring Cloud App Starters] currently have the metrics emitter module already bundled on it.
-You can enable metric emission by setting a flag explained http://docs.spring.io/spring-cloud-stream/docs/Chelsea.SR1/reference/htmlsingle/index.html#_metrics_emitter[here]
+You can enable metric emission by setting the destination name for metrics binding, e.g. `spring.cloud.stream.bindings.applicationMetrics.destination=<DESTINATION_NAME>`
+
+Starting with Spring Cloud Stream 2.0 the metrics has been switched to use https://micrometer.io/[Micrometer]. See https://docs.spring.io/spring-cloud-stream/docs/Elmhurst.RELEASE/reference/htmlsingle/#spring-cloud-stream-overview-metrics-emitter [Metrics Emitter] for more details
+Information how to configure the metrics for legacy Spring Cloud Stream 1.x applications is provided https://docs.spring.io/spring-cloud-stream/docs/Ditmars.SR3/reference/htmlsingle/#_metrics_emitter[here].
+
+The metrics collector 2.x can collect and process metrics coming from Spring Cloud Stream 1.x and 2.x applications.
```

Comment:
"from both Spring Cloud Stream 1.x and 2.x applications. "

Reply:
saying 'legacy' feels over the top....